processes/roles.md

```diff
@@ -30,7 +30,6 @@ You can nominate someone for a role by emailing the chair of the TSC.
   - Jesse Millwood, Dornerworks
```

Comment:
There is an affiliation change in the reviewers also, as far as I know Jesse has left Dornerworks some time ago.

Reply:
He's working for NASA Goddard Space Flight Center now, and left Dornerworks for Masten Space Systems in 2021 according to his Linkedin.